backend/schemas/user.py

```
@@ -7,7 +7,10 @@
 
 from datetime import datetime
 from typing import Optional
-from pydantic import BaseModel, EmailStr, Field, ConfigDict
+from pydantic import BaseModel, EmailStr, Field, validator, ConfigDict
+from typing import Optional
```

Ruff (0.13.2)

Redefinition of unused `Optional` from line 9

Remove definition: `Optional`

(F811)